database/engine/rrdengine.c

```diff
@@ -1084,6 +1101,7 @@ void timer_cb(uv_timer_t* handle)
             }
         }
     }
+    load_configuration_dynamic();
```

Comment:
I think it can be expensive to load the config all the time -- perhaps only when `cmd_write_config_execute` is called?

Reply:
In theory you are correct. In practice I don't think it's going to be a problem to have 2 AVL extra lookups per second.